packages/commonjs-shim/src/index.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2023.
+ * The core of this plugin was conceived by pi0 and is taken from the following repository:
+ * https://github.com/unjs/unbuild/blob/main/src/builder/plugins/cjs.ts
+ */
```

Comment:
good attempt, but we need to be a tad more proper about this. in the plugin's root directory we should have a copy of https://github.com/unjs/unbuild/blob/main/LICENSE named `unbuild.LICENSE` which contains the original text of that LICENSE. In the plugin's README in the `## Meta` section, we should include a line which states `Portions of this code were used with permission from [link to unbuild]. [unbuild LICENSE](./unbuild.LICENSE)`

If it was MPL or copyleft, we'd have to annotate our source.

```suggestion
```

Reply:
I have to be honest, I only followed the procedure that pi0 also follows for code that he integrates externally into his code, which he does not define as a depenency, but copies into his codebase.
We also have official permission from him to do whatever we want with the code, which is also covered by the MIT license. So I think our reference is more than sufficient and no explicit mention in the README.md or in the LICSENSE file is necessary.

Reference: 
- https://github.com/unjs/h3/blob/main/src/utils/multipart.ts
- https://github.com/unjs/h3/blob/main/LICENSE

Reply:
i like that argument :blush: it is always good to set a good example